core/container/src/main/java/org/wildfly/swarm/container/runtime/RuntimeServer.java

```diff
@@ -198,7 +200,9 @@ public ExtensibleConfigurationPersister createConfigurationPersister(ServerEnvir
 
         List<ServiceActivator> activators = new ArrayList<>();
 
-        this.serviceActivators.forEach(activators::add);
+        StreamSupport.stream(serviceActivators.spliterator(), false)
```

Comment:
Any particular reason for this usage vs `this.serviceActivators.stream()` ?

Reply:
`Instance::stream` is not available in CDI 1.2, it was added in 2.0 https://docs.jboss.org/cdi/api/2.0/javax/enterprise/inject/Instance.html#stream--

Reply:
Ah, I think I got confused with the `List` just above that code with a similar name. Forgot it was dealing with `Instance`.

Ignore me